PowerPointLabs/Test/UnitTest/TooltipsLab/CreateCalloutTest.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using PowerPointLabs.TooltipsLab;
+using PowerPointLabs.Models;
+
+using Test.Util;
+using Microsoft.Office.Interop.PowerPoint;
+
+namespace Test.UnitTest.TooltipsLab
+{
+    [TestClass]
+    public class CreateCalloutTest : BaseUnitTest
+    {
+        protected override string GetTestingSlideName()
+        {
+            return "TooltipsLab\\CreateCallout.pptx";
+        }
+
+        [TestMethod]
+        [TestCategory("UT")]
+        public void CreateCallout()
+        {
+            TestCreateCallout_NormalShape();
+            TestCreateCallout_TriggerShape();
+            TestCreateCallout_CalloutShape();
+            TestCreateCallout_MultipleNormalShapes();
+            TestCreateCallout_MultipleAllTypeShapes();
+            TestCreateCallout_MultipleTypeMoreShapes();
+        }
+
+        private const int CreateCalloutNormalShapeTestSlideNo = 4;
```

Comment:
Move these constants to on top all other functions and split the slides and names. Same for the other 2 unit tests!

Reply:
Done!